### PR TITLE
Added Navigation

### PR DESCRIPTION
**Pull request by furkanaskin**

### CODE REVIEW

**Review by mertceyhan — CHANGES_REQUESTED** [review]
You should open pull request as  "into gdgistanbul:develop from furkanaskin:#4-setup-navigation".

File: app/src/main/res/layout/main_activity.xml [hunk 1]
```diff
@@ -1,7 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/container"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity" />
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mertceyhan) [inline app/src/main/res/layout/main_activity.xml:2]: You don't need this, just can be fragment component

File: app/src/main/java/com/gdgistanbul/attendence/ui/user_list/UserListFragment.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * Created by Furkan on 10.07.2019
+ */
+
+class UserListFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
```
Comment (mertceyhan) [inline app/src/main/java/com/gdgistanbul/attendence/ui/user_list/UserListFragment.kt]: unnecessary comment

File: app/src/main/java/com/gdgistanbul/attendence/ui/main/MainFragment.kt [hunk 3]
```diff
@@ -29,12 +20,17 @@ class MainFragment : Fragment() {
...
         super.onViewCreated(view, savedInstanceState)
-        viewModel.eventsLiveData.observe(this, Observer {
-            message.text = "${it.size}"
-        })
-        viewModel.toastLiveData.observe(this, Observer {
-            Toast.makeText(context, it, Toast.LENGTH_LONG).show()
-        })
+
+        buttonSignInWithMeetup.setOnClickListener {
+            navigate(R.id.action_mainFragment_to_eventListFragment)
```
Comment (necatisozer) [inline app/src/main/java/com/gdgistanbul/attendence/ui/main/MainFragment.kt]: Better use action here

File: app/src/main/java/com/gdgistanbul/attendence/ui/user_list/UserListFragment.kt [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        Toast.makeText((activity as MainActivity), getEventID().toString(), Toast.LENGTH_SHORT).show()
+    }
+
+    private fun getEventID(): Int {
+        return UserListFragmentArgs.fromBundle(this.arguments!!).eventID
```
Comment (necatisozer) [inline app/src/main/java/com/gdgistanbul/attendence/ui/user_list/UserListFragment.kt]: You can use `by navArgs()` delegate
https://developer.android.com/guide/navigation/navigation-pass-data#Safe-args

**Pull request closed without merging**